atomic_reactor/util.py

```diff
@@ -1481,6 +1481,9 @@ class OSBSLogs(object):
     def __init__(self, log, platforms: list):
         self.log = log
         self.platforms = platforms
+        self.source_build = False
+        if not platforms:
+            self.source_build = True
```

Comment:
The problem is not the plugin itself, but the fact that it doesn't run at all. So the `get_platforms()` method gets None from the plugin results dict, because the key is not there.

Updating the method could be a good solution if it doesn't break anything

Reply:
updated the PR to just update `get_platforms` to return `[]` empty list